README.md

```diff
@@ -82,6 +82,20 @@ This flag could controls the settings of the undici client, like so:
 The number of parsed URLs that will be cached. Default: 100.
 > Use value = `0` to disable the caching mechanism
 
+#### requests.http and requests.https
+Override the internal `http` and `https` clients. Defaults: [`http`](https://nodejs.org/api/http.html#http_http) and [`https`](https://nodejs.org/api/https.html#https_https).
+
+This could be used to add support for following redirects, like so:
+
+```js
+...
+  requests: {
+    http: require('follow-redirects/http'),
+    https: require('follow-redirects/https')
+  }
+...
+```
+
```

Comment:
I wanted to give at least some idea of what this could be used for, even if the feature is flexible enough to be used in other ways.

By the way, currently I only support injecting `http` and `https`. I see that there is a function called `getHttp2` to use `http2` and `opts.unidici` to use that particular client. They presumably override this and don't need to be considered as part of this.

Reply:
This is fine for now. Thanks.